message_ix_buildings/chilled/util/util.py

```diff
@@ -22,26 +40,27 @@ def get_archs(config: "Config"):
 
             return archs
         else:
-            print(
+            raise TypeError(
                 "Archetypes input file "
                 + input_file
                 + " does not exist! Please create file for input."
             )
+
     elif config.arch_setting == "regional":
         input_file = os.path.join(
             version_path,
             "arch_input_reg.xlsx",
         )
 
         if os.path.exists(input_file):
-            archs = pd.read_excel(input_file, sheet_name="arch").arch.unique()
+            archs = pd.read_excel(input_file, sheet_name="arch").arch.unique()  # type: ignore
```

Comment:
Why do we need `# tpe: ignore` here?

Reply:
We don't -- I shall remove, thank you!

Reply:
Ah, without the `type: ignore` I was receiving a Mypy error: `

```
Incompatible types in assignment (expression has type "ndarray[Any, Any]", variable has type "list[int | str]").
``` 

I've removed the `type: ignore` and instead added a `.tolist()` to the end of the line, and that seems to have removed the linting error.